### ISSUE TITLE
Conflict on chrome.guid

### ISSUE THREAD

**Issue by pimterry** [vweevers/win-detect-browsers#67]
Hi @vweevers!

The previous fixes all seem to be doing great, which is excellent, but I have seen one automated report of a new issue:

```
Conflict on chrome.guid: "8A69D345-D564-463C-AFF1-A69D9E530F96" !== "4DC8B4CA-1BDA-483E-B5FA-D3C12E15B62D"
```

It's an anonymous report, and the user themselves hasn't got in touch, so I don't have any more info than the exception itself and I can't get a full debug trace. Any ideas what might cause this though, or how I might be able to reproduce it?

I've only seen this from one user, so it's pretty rare, but it'd be nice to chase it down and keep making this package more and more reliable in all these funky install environments if we can.

**Comment by vweevers** [comment]
I think the following understanding (of it being 32 and 64 bit) is incorrect or outdated:

https://github.com/vweevers/win-detect-browsers/blob/fa734ece4e992054a87e9095ef34a35406941097/lib/chrome/find-update-clients.js#L12-L13

[This](https://github.com/google/omaha/blob/ca928da3dc74a9bf4c5b952116f19def57749b17/CHANGELOG.txt#L695-L703) might provide a clue: 

> For first installs, Chrome uses {8A69D345-D564-463C-AFF1-A69D9E530F96} for installing. \[..\] For updates, Chrome uses {4DC8B4CA-1BDA-483e-B5FA-D3C12E15B62D}.

**Comment by vweevers** [comment]
From the [Chrome Browser Deployment Guide](https://docs.google.com/document/d/1iu6I0MhyrvyS5h5re5ai8RSVO2sYx2gWI4Zk4Tp6fgc/) (Google Docs):

> **Application Globally Unique Identifiers**
> When Chrome Browser is installed, it is entered into the Windows Registry with an Application Globally Unique Identifier (GUID). There will also be a parent GUID for the Chrome binaries. Any Registry changes you wish to make should be made to the parent GUID, especially as it relates to automatic updating.
>
> In Windows you can find these in the registry in these locations:
> **Chrome Binaries**
> {HKLM|HKCU}\Software\Google\Update\ClientState\{4DC8B4CA-1BDA-483E-B5FA-D3C12E15B62D}
> **Chrome**
{HKLM|HKCU}\Software\Google\Update\ClientState\{8A69D345-D564-463c-AFF1-A69D9E530F96}
>
> Important: The Chrome Binaries GUID will become the default for all installations. However, previous settings for an individual GUID will be maintained.

**Comment by vweevers** [comment]
TLDR, there is no real conflict. These 2 guids resolving to the same binary is legit.

Let's remove the `guid` property from the metadata, here:

https://github.com/vweevers/win-detect-browsers/blob/fa734ece4e992054a87e9095ef34a35406941097/lib/chrome/find-update-clients.js#L49-L52

As well as the `uninstall` property:

https://github.com/vweevers/win-detect-browsers/blob/fa734ece4e992054a87e9095ef34a35406941097/lib/chrome/find-update-clients.js#L62-L65

Seeing as there's no single truth for these (and I personally no longer need them).
